src/config.js

```diff
@@ -323,7 +358,7 @@ export const CONFIG = {
         .concat([
           'assorted-dom', 'ares6', 'octane', 'six-speed',
           'sunspider-jsbench', 'unity-webgl', 'wasm-misc',
-          'web-tooling']),
+          'wasm-godot', 'web-tooling']),
```

Comment:
I think you could make it a benchmark that is run for all platforms since I see it running for all platforms:
https://treeherder.mozilla.org/#/jobs?repo=mozilla-central&searchStr=godot

Reply:
Updated!